src/Sylius/Bundle/AdminBundle/templates/catalog_promotion/grid/field/date.html.twig

```diff
@@ -2,7 +2,7 @@
     {% if data is null %}
         <span class="text-secondary">-</span>
     {% else %}
-        <div>{{ data|date('Y-m-d') }}</div>
+        <div class="text-nowrap">{{ data|date('d-m-Y') }}</div>
         <div>{{ data|date('H:i') }}</div>
```

Comment:
```suggestion
        <div class="text-nowrap">{{ data|format_date }}</div>
        <div>{{ data|format_time }}</div>
```

Dates, times, currencies, locales display are different for each admin user locale.

Reply:
Yes I agree we could pass the format as field option with a default on the template

Reply:
But with date_format and time_format vars I think.

Reply:
We will change this format and make it consistent in the entire Admin in a separate PR, so this part has been reverted from this PR